### PR TITLE
[deliver] Support iMessage Screenshots

### PR DESCRIPTION
**Pull request by JustinJiaDev**
- [x] Run `bundle exec rspec` from the subdirectory of each tool you modified.
- [x] Run `bundle exec rubocop -a` to ensure the code style is valid.

Because iMessage screenshots and normal screenshots have same image resolution, we need to put all iMessage screenshots into an `iMessage` subdirectory. Currently `deliver` needs Apple TV screenshots be in the `appleTV` subdirectory, so I think this is the right approach.

This can be merged after #6823 being merged.

### CODE REVIEW

**Comment by KrauseFx** [comment]
Is this and https://github.com/fastlane/fastlane/pull/6823 ready for review and merge?

**Comment by JustinJiaDev** [comment]
@KrauseFx This and #6823 are ready now!

**Comment by milch** [comment]
[Ship iMessage Sticker Pack compatibility](https://trello.com/c/H1epLhZJ/533-ship-imessage-sticker-pack-compatibility)

**Comment by JustinJiaDev** [comment]
@milch Thanks for the link but seems like it's a private card. I can't view it.

File: deliver/lib/assets/ScreenshotsHelp [hunk 1]
```diff
@@ -1,5 +1,7 @@
 Put all screenshots you want to use inside the folder of its language (e.g. en-US).
 The device type will automatically be recognized using the image resolution. Apple TV screenshots
-should be stored in a subdirectory named appleTV with language folders inside of it.
+should be stored in a subdirectory named appleTV with language folders inside of it. iMessage
+screenshots, like Apple TV screenshots, should also be stored in a subdirectory name iMessage
```
Comment (milch) [inline deliver/lib/assets/ScreenshotsHelp]: `... in a subdirectory named...`

**Review by milch — CHANGES_REQUESTED** [review]
Seems solid as well! Just a small typo here.

**Comment by milch** [comment]
I tried doing some tests with this, and regular screenshot upload is still working fine (as expected), however, I couldn't get iMessage screenshots to upload. I created a folder titled `iMessage` in my `./fastlane/screenshots` folder and put the screenshots in the language-coded subfolder, e.g. `./fastlane/screenshots/iMessage/en-US/screen1.png`. Should there be any other steps necessary?

File: deliver/lib/assets/ScreenshotsHelp (hunk 1 above)
Comment (JustinJiaDev) [inline deliver/lib/assets/ScreenshotsHelp]: Fixed!

**Comment by JustinJiaDev** [comment]
@milch 

I tested using my personal account and I worked as expected. Did you test it with the updated version of `spaceship`? Are iMessage screenshots being collected?

**Comment by milch** [comment]
@JustinJiaDev I just reset the whole screenshots folder and re-did everything, it seems to work now. Maybe I had a typo somewhere, although it's weird that uploading the regular screenshots worked then. 

Anyway, this is a great PR, thanks a lot for your contribution! 🚀

**Review by milch — APPROVED** [review]

**Comment by JustinJiaDev** [comment]
Thanks for the review!

**Pull request merged**